app/controllers/api/Figures.scala

```diff
@@ -89,4 +89,24 @@ extends Silhouette[Login, CachedCookieAuthenticator] {
     Ok(Json.obj("error" -> false))
   }
 
+  /**
+   * Update the caption of an existing figure (id).
+   *
+   * @param id   The id of the figure.
+   *
+   * @return  OK / Failed
+   */
+
+  def updateCaption(id: String) = SecuredAction(parse.multipartFormData) { implicit request =>
```

Comment:
Same here: just `parse.json` is enough (if you don't upload multipart form data)